visualization_marker_tutorials/README.md

```diff
@@ -0,0 +1,265 @@
+# visualization_marker_tutorials
```

Comment:
Personally, I like that the document lives in the same repo as the code, but maybe we want to have the tutorial document live on [docs.ros.org](https://docs.ros.org/en/rolling/Tutorials.html).

@clalancette Is there a long-term plan for how to deal with tutorials like this? For context, I've asked @rebecca-butler to port several of the RViz tutorial packages to ROS 2 and the code lives in this repository but the tutorial articles for ROS 1 live here:  http://wiki.ros.org/rviz/Tutorials

Reply:
> @clalancette Is there a long-term plan for how to deal with tutorials like this?

I think it is best for these tutorials to live alongside the code that they reference.  We are now [building documentation](https://build.ros2.org/view/Rdoc/) for all packages released into Rolling.  Once this PR is merged and the package is released into Rolling, the docs should show up on https://docs.ros.org .